include/zephyr/drivers/espi.h

```diff
@@ -450,11 +462,18 @@ typedef int (*espi_api_flash_write)(const struct device *dev,
 				    struct espi_flash_packet *pckt);
 typedef int (*espi_api_flash_erase)(const struct device *dev,
 				    struct espi_flash_packet *pckt);
+
+/* eSPI interrupt control */
+typedef int (*espi_api_interrupt_conf)(const struct device *dev,
```

Comment:
Will the interrupt configure parameter be defined by vendor drivers or defined common to all drivers.  The HW may fire many different interrupts.

Reply:
@scottwcpg  Plan is to have some generic categories (eSPI bus events, VWs, ACPI, Port80, OOB, flash, KBC), but allow vendor specific bitmask (like pinctrl flags) so each can provide as much granularity as they see fit

Reply:
Since new eSPI interrupt  API may take more discussion move into separate PR